modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +254,146 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+// .Operator
+* The `s2i-nodejs` image cannot use the `nodejs:14-ubi8-minimal` image stream to perform S2I builds. Doing so produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message. There is no workaround at this time.
```

Comment:
```suggestion
* The `s2i-nodejs` image cannot use the `nodejs:14-ubi8-minimal` image stream to perform S2I builds. Doing so causes the following error message: `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127`. There is no workaround for this issue.
```
- Better not imply that there will ever be a workaround.

Reply:
Updated. Thanks.